lib/colorado_lottery.rb

```diff
@@ -0,0 +1,24 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    if contestant.age <= 18
```

Comment:
This causes a problem in the boolean logic, where this condition will always be hit first if we pass in a contestant that is 18. Remember when we're setting up boolean methods, we don't need control flow via an if/else statement, but rather the condition that will evaluate to true or false. In this case you could have gotten this to run with an AND statement:
`contestant.age >= 18 && contestant.game_interests.any? {|interest| interest == game.name}`